### PR TITLE
Add docs about testing filter expressions in the brower console.

### PR DESCRIPTION
**Pull request by mythmon**
These were originally written by @Osmose and put in a Gist: https://gist.github.com/Osmose/d59da84e9531487e5210cc0444f3eab9

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -74,7 +74,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+exclude_patterns = ['_build', '.*']
```
Comment (mythmon) [inline docs/conf.py]: This makes Sphinx ignore hidden files, which helps it work better in my workflow.

**Comment by andymikulski** [comment]
Would it be worth it to implement a little REPL into the site's front end to do this? It could handle `await`ing the promise response and all that, might make it more accessible. I can't imagine a ton of users would need/use it, though.

**Comment by mythmon** [comment]
I think a REPL would be a useful feature, but it would be difficult to expose the actual Shield implementation to the site's frontend. We talked about doing this in the past for the preview feature, and couldn't figure out a safe way to do it, and so backed off. I've been pondering if we could get away with some sort of extension to do this, but I haven't come up with anything solid yet.

It would be pretty simple to make a REPL that executed filter expressions in the webpage, but it wouldn't include the extra details added by the recipe client, such as Firefox version or Telemetry  pings.

**Comment by mythmon** [comment]
I think a REPL would be a useful feature, but it would be difficult to expose the actual Shield implementation to the site's frontend. We talked about doing this in the past for the preview feature, and couldn't figure out a safe way to do it, and so backed off. I've been pondering if we could get away with some sort of extension to do this, but I haven't come up with anything solid yet.

It would be pretty simple to make a REPL that executed filter expressions in the webpage, but it wouldn't include the extra details added by the recipe client, such as Firefox version or Telemetry  pings.

**Comment by mythmon** [comment]
I probably accidentally deleted this branch, and that's why this got closed. It is definitely something I still want.

**Pull request merged**